_data/categories/gw.yml

```diff
@@ -12,4 +12,22 @@
   desc: "Kong uses Nginx"
 - name: OpenResty
   link: http://openresty.org
-  desc: "OpenResty uses Nginx"
\ No newline at end of file
+  desc: "OpenResty uses Nginx"
+- name: sdsfdf
```

Comment:
@madhabpaul, it looks like lines 16, 17 and 18 need to be blanked out.

Reply:
yaa @leecalcote  16, 17, 18 need to be removed

Reply:
I have removed the lines.